build.gradle

```diff
@@ -13,6 +13,15 @@
 
 import net.ltgt.gradle.errorprone.CheckSeverity
 
+buildscript {
+  repositories { jcenter() }
+
+  dependencies {
+    classpath 'org.apache.openjpa:openjpa:3.1.0'
```

Comment:
Why not have these version in `versions.grade` with the other dependency versions?

Reply:
Done.
There wasn't one. But have added one.